package.json

```
@@ -68,6 +68,6 @@
   },
   "homepage": "https://github.com/prebuild/prebuild",
   "engines": {
-    "node": "^14.17.0 || ^16.13.0 || >=18.0.0"
+    "node": "^16.13.0 || >=18.0.0"
```

https://github.com/nodejs/node-gyp/pull/2929
```suggestion
    "node": "^16.14.0 || >=18.0.0"
```

---

Done